oak-search-elastic/src/test/java/org/apache/jackrabbit/oak/plugins/index/elasticsearch/ElasticsearchPropertyIndexTest.java

```diff
@@ -204,4 +213,32 @@ private String explain(String query) {
         String explain = "explain " + query;
         return executeQuery(explain, "JCR-SQL2").get(0);
     }
+
+    private static void assertEventually(Runnable r, long timeoutMillis) {
```

Comment:
Good! Adding a convenience method with just one parameter (Runnable) would be nice, because all uses currently use BULK_FLUSH_INTERVAL_MS_DEFAULT * 3 AFAIK.